website/www/site/layouts/partials/section-menu/en/get-started.html

```diff
@@ -37,16 +38,16 @@
   <span class="section-nav-list-title">Learning resources</span>
 
   <ul class="section-nav-list">
-    <li><a href="/documentation/resources/learning-resources/#getting-started">Getting Started</a></li>
```

Comment:
Should it cause an increase of 404 requests that will use old URLs (e.g. web search)? 
Should we add redirects in this case?

Reply:
Thank you for your point. Added an aliases to learning-resources.md, so now there is a redirect to correct path /get-started/resources/learning-resources even in case somebody uses /documentation/resources/learning-resources, so there will be no 404 error in case of old web search or others.

![urls](https://user-images.githubusercontent.com/61748758/190253962-237f02ec-58bc-4588-a4f4-f1410ffe1220.gif)

Reply:
Thanks!